FirebasePerformance/Sources/Configurations/FPRConfigurations.h

```diff
@@ -16,6 +16,15 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/** Different modes of prewarm-detection */
+typedef NS_ENUM(NSInteger, PrewarmDetectionMode) {
```

Comment:
nit: Can you add comments to the individual ENUMs here? I felt a bit lost when reviewing the code. It would have helped to have contextual reference on the value for these ENUMs